website/docs/d/alert_configuration.html.markdown

```diff
@@ -0,0 +1,217 @@
+---
+layout: "mongodbatlas"
+page_title: "MongoDB Atlas: alert_configuration"
+sidebar_current: "docs-mongodbatlas-datasource-alert-configuration"
+description: |-
+    Describes a Alert Configuration.
+---
+
+# mongodbatlas_alert_configuration
+
+`mongodbatlas_alert_configuration` describes an Alert Configuration.
+
+-> **NOTE:** Groups and projects are synonymous terms. You may find **group_id** in the official documentation.
+
+
+## Example Usage
+
+```hcl
+resource "mongodbatlas_alert_configuration" "test" {
+  project_id = "<PROJECT-ID>"
+  event_type = "OUTSIDE_METRIC_THRESHOLD"
+  enabled    = true
+
+  notification {
+    type_name     = "GROUP"
+    interval_min  = 5
+    delay_min     = 0
+    sms_enabled   = false
+    email_enabled = true
+  }
+
+  matcher {
+    field_name = "HOSTNAME_AND_PORT"
+    operator   = "EQUALS"
+    value      = "SECONDARY"
+  }
+
+  metric_threshold = {
+    metric_name = "ASSERT_REGULAR"
+    operator    = "LESS_THAN"
+    threshold   = 99.0
+    units       = "RAW"
+    mode        = "AVERAGE"
+  }
+}
+
+
+data "mongodbatlas_alert_configuration" "test" {
+	project_id             = "${mongodbatlas_alert_configuration.test.project_id}"
+	alert_configuration_id = "${mongodbatlas_alert_configuration.test.alert_configuration_id}"
+}
+```
+
+## Argument Reference
+
+* `project_id` - (Required) The ID of the project where the alert configuration will create.
+* `alert_configuration_id` - (Required) Unique identifier for the alert configuration.
+
+## Attributes Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `group_id` - Unique identifier of the project that owns this alert configuration.
+* `created` - Timestamp in ISO 8601 date and time format in UTC when this alert configuration was created.
+* `updated` - Timestamp in ISO 8601 date and time format in UTC when this alert configuration was last updated.
+* `enabled` - If set to true, the alert configuration is enabled.If enabled is not specified in a POST command, it defaults to false.
```

Comment:
Not sure POST makes sense here - data sources info?      Maybe change to:
If set to true, the alert configuration is enabled.  If enabled is not exported it is set to false.

Reply:
Thank you!